backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewToolService.java

```diff
@@ -496,7 +496,10 @@ public Element reconnnectTargetCompositionEdge(Element self, Element oldTarget,
         } else {
             var oldContainer = oldTarget.eContainer();
             if (newTarget instanceof Usage && oldContainer instanceof FeatureMembership featureMembership) {
-                // move the old target to the innermost package
+                // Move the old target to the innermost package
+                // We don't use moveToClosestContainingPackage here because we are moving oldTarget in the closest
+                // containing package
+                // and updating newTarget at the same time.
```

Comment:
the third line should probably be append on the second line

Reply:
done